src/components/Registration/Registration.tsx

```diff
@@ -21,9 +22,38 @@ const Registration: FC<RegistrationProps> = (props) => {
   const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     setPage(page + 1);
-    if(page === 4) {
+    if (page === 2) {
+      const userInfo = {
+        email,
+        password,
+        fullname: name,
+        bio,
+        make,
+        model,
+        year
+      }
+      postUserInfo(userInfo)
+        .then(response => setCurrentUserId(response.data.id))
+    }
+    if (page === 4) {
+
+      const routeData = {
+        user_id: currentUserId,
+        origin,
+        destination,
+        departure_time: departTime,
+        days: Object.keys(days).filter(el => days[el] === true)
+      }
+      console.log(routeData)
+      postRouteData(routeData)
+        .then(response => {
+          if (response.data.attributes) {
+            <Redirect to='/matched-routes' />
+          }
```

Comment:
This needs a tiny change and then all of our api calls should be working

Reply:
Good catch fixing this!